.pre-commit-config.yaml

```diff
@@ -1,9 +1,9 @@
-exclude: src/static/*|LICENSE.md|.+\.pem|.+\.crt|.vscode/*
+exclude: src/static/.+|.+\.pem|.+\.crt|.vscode/.+|LICENSE.md
 default_language_version:
   python: python3.8
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.3.0
+    rev: v4.4.0
```

Comment:
Different hooks may or may not have `latest` tags. Using `latest` tags would be inconsistent since some hooks don't provide them, and since breaking changes might appear without warning.

Since making the changes in this PR, [pre-commit CI](https://pre-commit.ci/) has been activated for this repository, which will auto-generate PRs to update pre-commit hooks when they are available. See an example of this type of PR [here](https://github.com/NTIA/scos-tekrsa/pull/38) - it's similar to using dependabot. You can also see that the pre-commit CI check is one of the ones which has passed for this PR.

Reply:
Fancy. I was mostly worried about if we could avoid a manual effort on that but it sounds like pre-commit covers that. Resolved.